tests/integration/jobs/verify_reports.rs

```diff
@@ -345,11 +384,12 @@ async fn setup_test(
                             ]
                         },
                         {
-                            "original": true,
+                            "original": false,
```

Comment:
This diff is a bit of a mess because I realized I had an unnecessary entry in here from earlier tests.